### ISSUE TITLE
Shellbar action button is missing Alt Text, also missing Aria-label when notificationCount is zero

### ISSUE THREAD

**Issue by itsnode** [SAP/fundamental-ngx#10956]
#### Is this a bug, enhancement, or feature request?
Bug
#### Can you handle that on the application side
no
#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
latest

#### Please provide stackblitz example(s).
https://stackblitz.com/edit/tw3jcd-bgufky?file=src%2Fapp%2Fshellbar-collapsible-example.component.html
